usaspending_api/broker/lookups.py

```diff
@@ -11,6 +11,7 @@
     LookupType(1, "fpds", "FPDS transactions from Broker"),
     LookupType(2, "fabs", "FABS transactions from Broker"),
     LookupType(3, "exec_comp", "Executive Compensation from Broker"),
+    LookupType(4, "zips", "Zip code data from broker"),
```

Comment:
Why was LookupTypeId 4 used?

Reply:
`4` was the next incremental value available. The values above `10` are grouped together by their types or uses. Since `zips` is a "standalone" type, I chose to use the next available value below 10.